lib/assessment.js

```diff
@@ -4,23 +4,39 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
+var min = function () {
+  var minimum = arguments[0]
+  for (i=1; i< arguments.length; i++){
+      if (arguments[i] < minimum) {
+        mimumun = arguments][i];
```

Comment:
Typos are some of the most insidious of bugs. They can break a lot of functionality and usually are too small to see without squinting. There is a typo on each side of the assignment operator (`=`) in this statement.